arch/rv32i/src/epmp.rs

```diff
@@ -108,6 +112,7 @@ impl<const MAX_AVAILABLE_REGIONS_OVER_TWO: usize> PMP<MAX_AVAILABLE_REGIONS_OVER
             // Reset back to how we found it
             csr::CSR.pmpconfig_set(i / 4, pmpcfg_og);
         }
+        */
```

Comment:
Tock doesn't usually like leaving commented-out code in the repository. They'd prefer that PR authors remove the code, and if it is needed later someone can dig it out of git history.